<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstDialog</class>
 <widget class="QDialog" name="FirstDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>1033</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>1011</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 185, 110);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="coke_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>810</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>36</string>
     </property>
    </widget>
    <widget class="QLabel" name="biscuit">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>320</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Parle-G</string>
     </property>
    </widget>
    <widget class="QLabel" name="coke_3">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>510</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtoncoke">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>580</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QLabel" name="bread">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>650</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Bread</string>
     </property>
    </widget>
    <widget class="QLabel" name="milk">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>650</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Amul Milk</string>
     </property>
    </widget>
    <widget class="QLabel" name="coke">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Coca Cola</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonbread">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>860</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonmilk">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>860</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonViewCart">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>930</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);
</string>
     </property>
     <property name="text">
      <string>View cart</string>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Umpush</family>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 121, 0);</string>
     </property>
     <property name="text">
      <string>     Welcome To Kirana Store</string>
     </property>
    </widget>
    <widget class="QLabel" name="coke_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>510</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>40</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonbiscuit">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>560</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QLabel" name="coke_5">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>800</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>40</string>
     </property>
    </widget>
    <widget class="QLabel" name="cokepic">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>390</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>coke</string>
     </property>
    </widget>
    <widget class="QLabel" name="parlepic">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>390</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>parle</string>
     </property>
    </widget>
    <widget class="QLabel" name="amulpic">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>700</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="breadpic">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>700</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelimgk">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>261</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="title_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Umpush</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 175, 62);
</string>
     </property>
     <property name="text">
      <string>  Items in Cart</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_itemlist">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>271</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 207);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBackToSelection">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>480</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonviewPay">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>480</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Pay</string>
     </property>
    </widget>
    <widget class="QLabel" name="total">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>420</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);
</string>
     </property>
     <property name="text">
      <string>Total :-</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalprice">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>420</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Umpush</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cartimg">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>570</y>
       <width>491</width>
       <height>311</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>300</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QTextEdit" name="textEditCardNum">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>390</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
    </widget>
    <widget class="QLabel" name="coke_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 175, 62);</string>
     </property>
     <property name="text">
      <string>Name :-</string>
     </property>
    </widget>
    <widget class="QLabel" name="total_pay">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 175, 62);</string>
     </property>
     <property name="text">
      <string>Total :-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonviewConfirm">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>450</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditName">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>330</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Umpush</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 175, 62);
</string>
     </property>
     <property name="text">
      <string>  Payment</string>
     </property>
    </widget>
    <widget class="QLabel" name="coke_9">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>390</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yrsa</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 175, 62);</string>
     </property>
     <property name="text">
      <string>Card No. :-</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalprice_Pay">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Umpush</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="paymentimg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>921</width>
       <height>591</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>paymentimg</zorder>
    <zorder>textEditCardNum</zorder>
    <zorder>coke_8</zorder>
    <zorder>total_pay</zorder>
    <zorder>pushButtonviewConfirm</zorder>
    <zorder>textEditName</zorder>
    <zorder>title_3</zorder>
    <zorder>coke_9</zorder>
    <zorder>totalprice_Pay</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>371</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 175, 62);</string>
     </property>
     <property name="text">
      <string>  Customer Bill</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>570</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="seedetails">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>570</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Generate Bill</string>
     </property>
    </widget>
    <widget class="QLabel" name="billimg">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>640</y>
       <width>421</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="bill2img">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>36</y>
       <width>911</width>
       <height>641</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>bill2img</zorder>
    <zorder>plainTextEdit</zorder>
    <zorder>label</zorder>
    <zorder>pushButton</zorder>
    <zorder>seedetails</zorder>
    <zorder>billimg</zorder>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QPushButton" name="pushButtonview_home">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>330</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>200</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Umpush</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 175, 62);
</string>
     </property>
     <property name="text">
      <string>Thank you..</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
